src/lib/apiTypesV2/Entity.cpp

```diff
@@ -237,12 +248,23 @@ void Entity::present(const std::string& indent)
 *
 * Entity::fill - 
 */
-void Entity::fill(const std::string& _id, const std::string& _type, const std::string& _isPattern, ContextAttributeVector* aVec)
+void Entity::fill
+(
+  const std::string&       _id,
+  const std::string&       _type,
+  const std::string&       _isPattern,
+  ContextAttributeVector*  aVec,
+  double                   _creDate,
+  double                   _modDate
```

Comment:
I'm just using the same that was used before this PR... See for example https://github.com/telefonicaid/fiware-orion/pull/2590/files#diff-41b80b5423da4a52cade90d9124bcb87L51

Changing to `long long` could be a better option... but we need to analyze the implications (including the ones at DB level). I'd suggest to open an issue about it.